src/style.css

```diff
@@ -1 +1,25 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
Consider using a fallback font-family to ensure text is displayed with a reasonable font even if the main font isn't available.